staging/src/k8s.io/client-go/util/flowcontrol/throttle_test.go

```diff
@@ -62,11 +62,11 @@ func TestMultithreadedThrottling(t *testing.T) {
 	close(startCh)
 	// wait for the first signal that we collected 100 tokens
 	<-endCh
-	// record wall time
-	endTime := time.Now()
+	// record duration since wall time
+	duration := time.Since(startTime)
```

Comment:
time.Since(startTime) and time.Now().Sub(startTime) are equivalent

https://golang.org/pkg/time/#hdr-Monotonic_Clocks